File: Programs/fizzbuzz.rb
b = 100
while b > 1
  b -= 1
  if (b%3!=0) and (b%5!=0)
    puts b 
  elsif (b%3==0) and (b%5==0)
    puts 'fizzbuzz ' + b.to_s  
  elsif b%3==0
    puts 'fizz ' + b.to_s 
  elsif b%5==0
    puts 'buzz ' + b.to_s
  end
end

File: Programs/fizzbuzz.js
// every 3rd fizz, every 5th buzz every 15th fizzbuzz

for(i=0;i<101;i++){
	switch(i) {
		case (i%3==0):
			i = "fizz";
		case (i%5==0):
			i = "buzz";
		case (i%3==0) && (i%5==0):
			i = "fizzbuzz";
		default:
			i=i;
	}
}